sdk/include/psdk/powrprof.h

```diff
@@ -8,91 +8,150 @@
 #include <ntsecapi.h>
 #endif
 
+#ifdef __REACTOS__
+typedef enum _POWER_PLATFORM_ROLE {
+    PlatformRoleUnspecified,
+    PlatformRoleDesktop,
+    PlatformRoleMobile,
+    PlatformRoleWorkstation,
+    PlatformRoleEnterpriseServer,
+    PlatformRoleSOHOServer,
+    PlatformRoleAppliancePC,
+    PlatformRolePerformanceServer,
+    PlatformRoleSlate,
+    PlatformRoleMaximum
+} POWER_PLATFORM_ROLE, *PPOWER_PLATFORM_ROLE;
+
+#endif
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-#define EnableMultiBatteryDisplay 2
-#define EnablePasswordLogon 4
-#define EnableSysTrayBatteryMeter 1
-#define EnableWakeOnRing 8
-#define EnableVideoDimDisplay 16
-#define NEWSCHEME (UINT)-1
-
-#ifndef RC_INVOKED
-
-typedef struct _GLOBAL_MACHINE_POWER_POLICY{
-   ULONG Revision;
-   SYSTEM_POWER_STATE LidOpenWakeAc;
-   SYSTEM_POWER_STATE LidOpenWakeDc;
-   ULONG BroadcastCapacityResolution;
-} GLOBAL_MACHINE_POWER_POLICY, *PGLOBAL_MACHINE_POWER_POLICY;
-typedef struct _GLOBAL_USER_POWER_POLICY{
-   ULONG Revision;
-   POWER_ACTION_POLICY PowerButtonAc;
-   POWER_ACTION_POLICY PowerButtonDc;
-   POWER_ACTION_POLICY SleepButtonAc;
-   POWER_ACTION_POLICY SleepButtonDc;
-   POWER_ACTION_POLICY LidCloseAc;
-   POWER_ACTION_POLICY LidCloseDc;
-   SYSTEM_POWER_LEVEL DischargePolicy[NUM_DISCHARGE_POLICIES];
-   ULONG GlobalFlags;
-} GLOBAL_USER_POWER_POLICY, *PGLOBAL_USER_POWER_POLICY;
-typedef struct _GLOBAL_POWER_POLICY{
-   GLOBAL_USER_POWER_POLICY user;
-   GLOBAL_MACHINE_POWER_POLICY mach;
-} GLOBAL_POWER_POLICY, *PGLOBAL_POWER_POLICY;
-typedef struct _MACHINE_POWER_POLICY{
-   ULONG Revision;
-   SYSTEM_POWER_STATE MinSleepAc;
-   SYSTEM_POWER_STATE MinSleepDc;
-   SYSTEM_POWER_STATE ReducedLatencySleepAc;
-   SYSTEM_POWER_STATE ReducedLatencySleepDc;
-   ULONG DozeTimeoutAc;
-   ULONG DozeTimeoutDc;
-   ULONG DozeS4TimeoutAc;
-   ULONG DozeS4TimeoutDc;
-   UCHAR MinThrottleAc;
-   UCHAR MinThrottleDc;
-   UCHAR pad1[2];
-   POWER_ACTION_POLICY OverThrottledAc;
-   POWER_ACTION_POLICY OverThrottledDc;
-} MACHINE_POWER_POLICY, *PMACHINE_POWER_POLICY;
+#define EnableSysTrayBatteryMeter	0x01
+#define EnableMultiBatteryDisplay	0x02
+#define EnablePasswordLogon		0x04
+#define EnableWakeOnRing		0x08
+#define EnableVideoDimDisplay		0x10
+
+typedef struct _GLOBAL_MACHINE_POWER_POLICY {
+	ULONG Revision;
```

Comment:
Please restore the 2 or 4-space indentation.

Reply:
This is a reduced diff with WINE.